source/widgets/ProjectStatus/js/configuration.js

```diff
@@ -4,6 +4,7 @@ function OctopusStatusWidgetConfiguration() {
         VSS.register("OctoProjectEnvironmentWidget.Configuration", function () {
             var $connectionDropdown = $("#octopus-connection");
             var $spaceDropdown = $("#octopus-space");
+            var $projectFilterInput = $("#octopus-project-filter");
```

Comment:
wowzers, jquery. That takes me back. I'd blocked out all the trauma of the og `vss-web-extension-sdk` limitations - you're a trooper 😂  

It's probably a bit much for this PR, but as some point we should update this whole extension to the newer `azure-devops-extension-sdk` and leverage the react ui library. That would allow us to use searchable dropdowns [like this one](https://developer.microsoft.com/en-gb/azure-devops/components/dropdown#advanced-dropdown)

Reply:
I had to re-remember that `.val()` serves as both a getter and setter as well. Yea, the memories of 10+ years ago.

Reply:
Yes completely agree, I want us to make a separate card in QRF to revisit this and see if we can sell a revamp using the new libraries / get with the times. That searchable dropdown is exactly what we needed in this case.